src/compressed_tensors/quantization/lifecycle/initialize.py

```diff
@@ -82,7 +83,7 @@ def _initialize_scale_zero_point_observer(
 ):
     # initialize observer module and attach as submodule
     observer = quantization_args.get_observer()
-    module.register_module(f"{base_name}_observer", observer)
+    module.register_module(f"{base_name}_{ObserverTypes.BASE.value}", observer)
```

Comment:
small nit - do we need to inherit from enum if we're only going to be referencing the value? (just to avoid the extra .value call)